<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Backward">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play_Pause">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Forward">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>510</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Current_Time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_A_Video_File">
    <property name="title">
     <string>Select A Video File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuSelect_A_Video_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
